server/msteams/client.go

```diff
@@ -463,7 +478,13 @@ func (tc *ClientImpl) SubscribeToChannels(baseURL string, webhookSecret string,
 	return tc.subscribe(baseURL, webhookSecret, resource, changeType)
 }
 
-func (tc *ClientImpl) SubscribeToChats(baseURL string, webhookSecret string, pay bool) (string, error) {
+func (tc *ClientImpl) SubscribeToChannel(teamID, channelID, baseURL string, webhookSecret string) (*Subscription, error) {
+	resource := fmt.Sprintf("/teams/%s/channels/%s/messages", teamID, channelID)
```

Comment:
Doesn't this resource require the license model?

Reply:
Nope, actually, it is not supported, AFAIK